go-controller/pkg/metrics/master.go

```diff
@@ -433,7 +433,11 @@ func RunTimestamp(stopChan <-chan struct{}, sbClient, nbClient libovsdbclient.Cl
 
 // RecordPodCreated extracts the scheduled timestamp and records how long it took
 // us to notice this and set up the pod's scheduling.
-func RecordPodCreated(pod *kapi.Pod) {
+func RecordPodCreated(pod *kapi.Pod, netInfo util.NetInfo) {
+	if netInfo.IsSecondary() {
+		// TBD: no op for secondary network for now, TBD
+		return
+	}
```

Comment:
this will need revisit once we decide how to implement the secondary network metrics. We may need different implementation or for some cases, we may not need it at all for the secondary network. My thinking is that to keep those metrics related function the way as today so that we don't miss any of them in the future.